scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/PPMLContext.scala

```diff
@@ -89,6 +91,10 @@ class PPMLContext protected(kms: KeyManagementService, sparkSession: SparkSessio
   def write(dataFrame: DataFrame, cryptoMode: CryptoMode): EncryptedDataFrameWriter = {
     new EncryptedDataFrameWriter(sparkSession, dataFrame, cryptoMode, dataKeyPlainText)
   }
+
+  def getSparkSession(): SparkSession = {
```

Comment:
scala doc

Reply:
Will fix in https://github.com/intel-analytics/BigDL/pull/4783